### ISSUE TITLE
iocage chroot doesn't work with hack88=1

### ISSUE THREAD

**Issue by cactus** [iocage/iocage#122]
```
$ zfs list -r xyz/iocage/jails
xyz/iocage/jails/5bf845ee-1ebd-11e5-9ed8-0cc47a31451a            4.16G   205G   100K  /iocage/jails/5bf845ee
xyz/iocage/jails/5bf845ee-1ebd-11e5-9ed8-0cc47a31451a/root       4.16G   205G  4.47G  /iocage/jails/5bf845ee/root

$ sudo iocage chroot 5bf845ee
chroot: /iocage/jails/5bf845ee-1ebd-11e5-9ed8-0cc47a31451a/root: No such file or directory
```

**Comment by pannon** [comment]
@cactus thanks for reporting this, latest version in `develop` should be all good. Give it a try.

**Comment by skarekrow** [comment]
Yes, fixed in latest commit fad17e93f8. @cactus please reopen if this does not work for you.

**Comment by cactus** [comment]
:+1: develop is working great. Thanks!
